config/initializers/devise.rb

```diff
@@ -265,9 +265,9 @@
 end
 
 Rails.application.config.to_prepare do
-  Devise::SessionsController.layout "devise"
-  Devise::RegistrationsController.layout proc{ |controller| user_signed_in? ? "application" : "devise" }
-  Devise::ConfirmationsController.layout "devise"
-  Devise::UnlocksController.layout "devise"
-  Devise::PasswordsController.layout "devise"
+  Devise::SessionsController.layout "admin/devise"
+  Devise::RegistrationsController.layout proc{ |controller| user_signed_in? ? "admin/signed_in" : "admin/devise" }
+  Devise::ConfirmationsController.layout "admin/devise"
+  Devise::UnlocksController.layout "admin/devise"
+  Devise::PasswordsController.layout "admin/devise"
```

Comment:
Devise is coupled to the admin system currently which is why I put the devise template stuff under 'admin'. If we end up needing to use devise for the crowd funding platform, we can refactor.

Reply:
This sounds sensible. Thank you.